packages/@uppy/core/src/index.js

```diff
@@ -428,7 +414,46 @@ class Uppy {
    */
   getFiles () {
     const { files } = this.getState()
-    return Object.keys(files).map((fileID) => files[fileID])
+    return Object.values(files)
```

Comment:
Is this on purpose?

Reply:
Yes, it does the same, but simpler

Reply:
I think I avoided this because of IE, but for 2.0 should be fine.

Reply:
I figured as much :)